.github/ISSUE_TEMPLATE/z-new-teammate-onboarding.md

```diff
@@ -82,6 +82,8 @@ Welcome to Fleet! This issue tracks TODOs required for all new teammates for you
   - If this is an engineer: [invite to "Engineering" team to grant full write access](https://github.com/orgs/fleetdm/teams/engineering/members)
   - Otherwise: [invite to "Teammates" team](https://github.com/orgs/fleetdm/teams/teammates/members)
   - [ ] Ask new teammate to set their membership visibility to public, if they are comfortable doing so
+  - [ ] Ask new teammate to bookmark Public Issues (https://github.com/fleetdm/fleet/issues)
+  - [ ] Ask new teammate to bookmark Confindential Issues (https://github.com/fleetdm/confidential/issues)
```

Comment:
Looks like this link to confidential issues is resulting in a 404 and triggering the failed markdown link check. I think this is because it's private repo. I can go ahead and add this link as an exception to the markdown check.